cds-snc/c19-benefits-node
# Remove cerb exhausted

# Description

Remove cerb exhausted option from cerb exhausted question. Add cypress tests for CRB, CRCB and CRSB benefit cards. 

## Test Instructions

1. run application and confirm the option does not exist on question 
2. run cypress tests



## Checklist
- [ N/A] Update CHANGELOG.md
- [ x] Unit tests have been added/updated
- [ x] e2e tests have been added/updated
